log.log

```diff
@@ -0,0 +1,234 @@
+b2e1143d was Candace Savonen, 22 hours ago, message: Merge branch 'master' into cansavvy/CONTRIB-staging
```

Comment:
I assume this add is not intentional

Reply:
Shoot. That's why I added `git.log` to gitignore. But I forgot first try I named it something else.